src/harness.js

```diff
@@ -271,18 +269,7 @@ class Harness {
    */
   async flush(amount) {
     let now = this._now || Date.now();
-    if (typeof amount === 'object') {
-      const { years = 0, days = 0, hours = 0, minutes = 0, seconds = 0, ms = 0 } = amount;
-      now = now +
-        years   * 1000 * 60 * 60 * 24 * 365 +
-        days    * 1000 * 60 * 60 * 24 +
-        hours   * 1000 * 60 * 60 +
-        minutes * 1000 * 60 +
-        seconds * 1000 +
-        ms;
-    } else { // shorthand is for days
-      now += amount * 24 * 60 * 60 * 1000;
-    }
+    now = addDate(now, amount);
```

Comment:
seems this `addDate` function is only used once. it is also only one line. I'd recommend removing it from the utils and just having the line here

Reply:
Aiming for separation of concerns such that date implementation is isolated in dateUtils and if we may need to update it later on, it can be done in one place

Reply:
my opinion is that Luxon should be our "date utility" for 99%+ of functionality.  i hope we can get out of the business of providing bespoke date utilities or providing any custom date interfaces, etc.  luxon does a better job than we can hope to achieve

this dateUtils is largely for backward compatibility for the old interface